docs/conf.py.in

```diff
@@ -160,6 +160,11 @@ examples_types = {
                 'lang': 'java',
                 'group': 'java'
         },
+        '^<examples>.*pythonic.*\.py$': {
+                'title': 'Python (pythonic)',
+                'lang': 'python',
+                'group': 'py-regular'
```

Comment:
Why is this `py-regular` and before it was `py-pythonicapi`?

Reply:
That is a mistake. Good catch!